### PR TITLE
Shorten name of new template

### PR DESCRIPTION
**Pull request by luciddreamz**
@bparees Please merge

### CODE REVIEW

File: openshift/templates/rails-postgresql-persistent.json [hunk 1]
```diff
@@ -2,15 +2,15 @@
   "kind": "Template",
   "apiVersion": "v1",
   "metadata": {
-    "name": "rails-postgresql-persistent",
+    "name": "rails-pgsql-persistent",
```
Comment (bparees) [inline openshift/templates/rails-postgresql-persistent.json:5]: we can't change the template name itself.

Comment (bparees) [reply openshift/templates/rails-postgresql-persistent.json:5]: also why is this necessary?

**Review by bparees — CHANGES_REQUESTED** [review]

File: openshift/templates/rails-postgresql-persistent.json (hunk 1 above)
Comment (sspeiche) [inline openshift/templates/rails-postgresql-persistent.json]: I thought we were waiting for the right "label" for display name to use, then we'd update with that. Pending approval/agreement on what that is from @smarterclayton

Comment (bparees) [reply openshift/templates/rails-postgresql-persistent.json:5]: discussed on irc, apparently this name is too long to be installable (more than 24 chars)

**Pull request merged**